grapher/core/Grapher.tsx

```diff
@@ -544,6 +546,14 @@ export class Grapher
             : this.chartInstanceExceptMap
     }
 
+    get facetYAxisMode(): FacetScaleMode {
+        return this.yAxis.facetScaleMode || FacetScaleMode.absolute
+    }
+
+    set facetYAxisMode(mode: FacetScaleMode) {
+        this.yAxis.facetScaleMode = mode
+    }
+
```

Comment:
Instead of creating these in order to implement the manager interface, could the `FacetYAxisToggleManager` be implemented by the `AxisConfig` instead? I think the `ScaleSelectorManager` is implemented similarly: https://github.com/owid/owid-grapher/blob/9bc2033bbdc0e834d0d8589a080232d602e6f84e/grapher/axis/AxisConfig.ts#L27

Not sure if it's definitely a good idea, since maybe only the Y axis should be toggle-able for now. But maybe for ScatterPlots we would like to control both, at some point? (Again, something along the same lines as the scale selector (log/linear switch))

Reply:
Good call, would prefer more stuff isn't implemented directly onto Grapher. I'll look at grafting it across.

Reply:
Done